installer

```diff
@@ -0,0 +1,131 @@
+#!/usr/local/cpython-3.4/bin/python3
```

Comment:
Too strict probably, and for consistency with other tools should be `#!/usr/bin/env python3`, and the filename should have the `.py` extension, and perhaps even be placed in tools/ directory though that makes it harder to discover.

Also the file should be formatted using tools/autoformat.py so that one might need to be changed as well (didn't test though).

Reply:
I do a specific #! sometimes, to document what version of Python I was testing against.  I agree that using /usr/bin/env is better for portability, even though it's less clear, and shows up in top less nicely.